test/azul_test_case.py

```diff
@@ -349,3 +352,15 @@ def setUp(self):
                 with self.assertRaises(KeyboardInterrupt):
                     TC(events).run()
                 self.assertEqual(events, expected_events)
+
+
+class AzulMotoTestCase(TestCase):
+    """
+    A mixin for tests that utilize the moto module
+    """
+
+    def setUp(self) -> None:
+        super().setUp()
+        # Moto backends are reset to ensure no residual resources are left over if
```

Comment:
```suggestion
        # Moto backends are reset to ensure no resources are left over if
```

Residual means left-over.